src/main/resources/swagger.api/responses/notFoundResponse.yaml

```diff
@@ -0,0 +1,9 @@
+description: Not found
+content:
+  application/json:
+    example:
+      errors:
+        - message: Request not found
```

Comment:
If this is meant to represent  a generic 404 response, I don't think we should mention "request" here